package.json

```diff
@@ -53,6 +57,8 @@
     "build": "react-scripts build",
     "dev": "cat .env.common-local .env.dev > .env.local && yarn start-dev",
     "eject": "react-scripts eject",
+    "eslint": "eslint --ext js --ext tsx src/",
```

Comment:
I think we also need `--ext ts` for declaration files, right?

Reply:
We do indeed. However, there is a meta problem: `amo/only-tsx-files` doesn't seem to fail for the *existence* of a `.ts` file. D'oh.

```
$ eslint --ext js --ext tsx --ext ts src/

/Users/kumar/dev/addons-code-manager/src/@types/react-diff-view/index.d.ts
   1:1   error  Only use `.tsx` file extensions in a TypeScript project                        amo/only-tsx-files
  16:14  error  Array type using 'Array<ChangeInfo>' is forbidden. Use 'ChangeInfo[]' instead  @typescript-eslint/array-type
  25:16  error  Array type using 'Array<HunkInfo>' is forbidden. Use 'HunkInfo[]' instead      @typescript-eslint/array-type
  40:55  error  Array type using 'Array<DiffInfo>' is forbidden. Use 'DiffInfo[]' instead      @typescript-eslint/array-type
  50:9   error  'Diff' is defined but never used                                               @typescript-eslint/no-unused-vars
  50:22  error  'React' is not defined                                                         no-undef

/Users/kumar/dev/addons-code-manager/src/react-app-env.d.ts
  1:1  error  Expected exception block, space or tab after '//' in comment  spaced-comment
  2:1  error  Only use `.tsx` file extensions in a TypeScript project       amo/only-tsx-files

/Users/kumar/dev/addons-code-manager/src/setupTests.ts
   1:1   error  Only use `.tsx` file extensions in a TypeScript project                                        amo/only-tsx-files
   1:1   error  'enzyme' should be listed in the project's dependencies, not devDependencies                   import/no-extraneous-dependencies
   2:1   error  'enzyme-adapter-react-16' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   3:1   error  'jest-fetch-mock' should be listed in the project's dependencies, not devDependencies          import/no-extraneous-dependencies
   4:1   error  'jest-enzyme' should be listed in the project's dependencies, not devDependencies              import/no-extraneous-dependencies
  10:1   error  Expected 1 empty line after require statement not followed by another require                  import/newline-after-import
  10:22  error  'jest-fetch-mock' should be listed in the project's dependencies, not devDependencies          import/no-extraneous-dependencies
  14:3   error  'fetchMock' is not defined                                                                     no-undef

✖ 16 problems (16 errors, 0 warnings)
```

Reply:
> amo/only-tsx-files doesn't seem to fail for the existence of a .ts file. 

Actually, wait, I think just adding `--ext ts` will address this

Reply:
arf, fixed.